backend/dataall/modules/datasets/handlers/glue_column_handler.py

```diff
@@ -38,7 +38,7 @@ def update_table_columns(engine, task: models.Task):
 
             aws_session = SessionHelper.remote_session(table.AWSAccountId)
 
```

Comment:
minor suggestion: define params`table=table, aws_session=aws_session` to avoid mistakes